frontend/src/modules/Organizations/components/OrganizationTeamInviteEditForm.js

```diff
@@ -47,35 +144,31 @@ export const OrganizationTeamInviteEditForm = (props) => {
           of. All members of that group will be able to access your environment.
```

Comment:
nit: edit this comment. All members of that group will inherit the permissions .....

Reply:
done